cmdx/config.go

```diff
@@ -75,6 +76,21 @@ func (c *Config) Load(cfg interface{}) error {
 	return nil
 }
 
+func (c *Config) LoadWithCmdBindFlag(cmd *cobra.Command, cfg interface{}) error {
```

Comment:
Shall we create a `WithFlags` midifier functions instead of a fresh loader similar to what we do in main config package? Do we see any cons of that?

Reply:
I was thinking to leave the Load function as is. But I think it is okay to create option as you suggest.

Reply:
@ravisuhag done with the update, I add opts to the `Load` function assuming that `Load` should always be called in every client cmd and the `WithFlag` require `cmd` info, to which `cmd` does the bind flag applied to.